src/ui/main.qml

```diff
@@ -63,33 +63,11 @@ Window {
             minimumWidth = Theme.desktopAppWidth
         }
 
-        Glean.initialize('MozillaVPN', VPNSettings.gleanEnabled && VPNFeatureList.get("glean").isSupported, {
+        Glean.setLogPings(true);
+        Glean.setDebugViewTag("vpn-testing");
```

Comment:
This tag is [a debugging feature](https://mozilla.github.io/glean/book/reference/debug/debugViewTag.html). Setting this means all pings will be tagged upon sending, in practice that means the pings are re-routed to Glean's debug ping viewer. You can see the ping I sent when testing this here: https://debug-ping-preview.firebaseapp.com/pings/vpn-testing